ui/templates/layout.html

```diff
@@ -5,19 +5,25 @@
 {{template "head" .Head}}
 
 <body>
-    <header class="navbar navbar-expand-lg flex-column flex-md-row navbar-dark tls-navbar-dark">
+    <header class="navbar navbar-expand-lg flex-column flex-md-row navbar-light">
         <a class="navbar-brand" href='#home'>
             <img src="/static/images/tlsential_web_logo.svg" alt="TLSential" height="42">
         </a>
         <div class="collapse navbar-collapse" id="navbar">
             <div class="navbar-nav" id="boq-navbar-links">
-                <a class="nav-item nav-link" href="dashboard.html">Dashboard</a>
+                <a class="nav-item nav-link" href="/ui/dashboard">Dashboard</a>
+            </div>
+            <div class="navbar-nav" id="boq-navbar-links">
+                <form action="/ui/logout" method="POST">
+                 {{ .CSRFField }}
+                <button class="nav-item nav-link" type="submit">Logout</label>
```

Comment:
Should be `<button ...>...</button>` not `</label>`

Reply:
Browsers are so forgiving nowadays.